MCU_Microchip_SAMD.lib

```diff
@@ -0,0 +1,146 @@
+EESchema-LIBRARY Version 2.4
+#encoding utf-8
+#
+# SAMD09C13A-SS
+#
+DEF SAMD09C13A-SS U 0 20 Y Y 1 F N
+F0 "U" -400 650 50 H V L CNN
+F1 "SAMD09C13A-SS" 50 650 50 H V L CNN
+F2 "Package_SO:SOIC-14_3.9x8.7mm_P1.27mm" 0 -1050 50 H I C CNN
+F3 "" 0 -700 50 H I C CNN
+ALIAS SAMD10C13A-SS SAMD10C14A-SS SAMD11C14A-SS
```

Comment:
Just my 2 cents: as there are some significant differences between the SAMD{09|10|11} family's pin functions (for example SAMD11 has USB pins), i think it would be nice to have these indicated on the symbols as well, rather than simply have aliases to the SAMD09 series

Reply:
guess you are right here. will try to find some time to figure the differences out and use different symbols.